src/System.Private.ServiceModel/tests/Scenarios/Client/ExpectedExceptions/ExpectedExceptionTests.cs

```diff
@@ -152,7 +154,7 @@ public static void SendTimeout_For_Long_Running_Operation_Throws_TimeoutExceptio
             {
                 IWcfService proxy = factory.CreateChannel();
                 watch.Start();
-                proxy.EchoWithTimeout("Hello");
+                proxy.EchoWithTimeout("Hello", serviceOperationTimeout);
             });
         }
         finally
```

Comment:
BTW, this test fails sporadically on NET Native with a time **less** than 5 seconds (like 4992 ms).  I don't know whether we should consider that a product bug, but we could make the test more reliable by allowing some grace period (like the elapsed time should be in the range 4000 ms to 8000 ms).